packages/dart/sshnoports/lib/src/print_devices.dart

```diff
@@ -26,5 +22,14 @@ void printDeviceList(Iterable<String> devices, Map<String, dynamic> info) {
   for (var device in devices) {
     stderr.writeln('  $device - v${info[device]?['version']}'
         ' (core v${info[device]?['corePackageVersion']})');
+    if (info[device]['allowedServices'] != null &&
+        (info[device]['allowedServices'] is List<String>) &&
+        (info[device]['allowedServices'] as List<String>).isNotEmpty) {
+      stderr.write("  - allowedServices:");
+      for (String service in info[device]['allowedServices']) {
+        stderr.write(' $service');
+      }
+      stderr.writeln();
+    }
```

Comment:
The format should be something like
```
Available devices:
  mydevice - v5.1.0 (Core v6.x.x)
  - allowedServices: localhost:22 localhost:3389
```

Reply:
Should be easy to parse, if the first non-whitespace character on the following line is a `-`, then this is an attribute associated with the device. `-` is not allowed in the device name, so this should be an acceptable approach.

I could change the `:` after 'allowedServices' since it conflicts with the `:` in the host:port scheme, but this can still even be parsed in the shell with cut, awk or sed.